utils/storage.js

```diff
@@ -37,7 +37,7 @@ function setDonations(donations) {
 module.exports = {
     getUsers,
     setUsers,
-    getIdeas,
+    getIdeasUtil: getIdeas,
```

Comment:
You can export `getIdeas` here - and then rename it when you import it in other functions

Reply:
Oh, but is it wrong to export it that way? ) 
To be honest I've had some troubles with renaming during import.

Reply:
Why is it wrong to export `getIdeas` as `getIdeas`?

Check documentation on how to rename during import - it should be straightforward

Reply:
Yeah, I think I've got it. Case solved )